pom.xml

```diff
@@ -12,12 +12,13 @@
   <artifactId>jaxen</artifactId>
   <packaging>bundle</packaging>
   <name>jaxen</name>
-  <version>1.1.6</version>
+  <version>1.1.7-SNAPSHOT</version>
   <description>Jaxen is a universal Java XPath engine.</description>
   <url>http://www.cafeconleche.org/jaxen</url>
   <licenses>
     <license>
-      <url>http://www.cafeconleche.org/jaxen/license.html</url>
+      <name>BSD-License</name>
+      <url>https://raw.githubusercontent.com/jaxen-xpath/jaxen/master/LICENSE.txt</url>
```

Comment:
guess we lost that when the site moved. I'm not sure that's actually accurate. I'll have to check.